### PR TITLE
Updated the documentation

### PR DESCRIPTION
**Pull request by raman08**
Issue: #233 

- Added the SWOC Lable
SWOC

- Changed the styling of the team quote.
Team

### CODE REVIEW

File: docs/gs.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<h1 align="center"><b>Getting Started...</b></h1> 
+<h1 align="center"><b>Getting Started...</b></h1>
```
Comment (smaranjitghose) [inline docs/gs.md:1]: In the last PR also,  I pointed out please use markdown specific syntax for Headings

File: docs/hosting.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+<h1 align="center"><b>Hosting your Postfolio On Cloud </b></h1>
```
Comment (smaranjitghose) [inline docs/hosting.md:1]: As suggested earlier

File: docs/hosting.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+## Netlify
+
+![netlify1](images/netlify.png)
```
Comment (smaranjitghose) [inline docs/hosting.md:5]: Rename the images properly please. put it inside ``images/hosting/``

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: docs/hosting.md (hunk 3 above)
Comment (raman08) [reply docs/hosting.md:5]: Okk sir

**Comment by smaranjitghose** [comment]
@raman08 Updates?

**Pull request closed without merging**